kie-spring-boot/kie-spring-boot-samples/jbpm-spring-boot-sample-basic/src/test/java/org/jbpm/springboot/samples/listeners/CountDownLatchNotificationListener.java

```diff
@@ -0,0 +1,44 @@
+package org.jbpm.springboot.samples.listeners;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.CountDownLatch;
+
+import org.jbpm.services.task.deadlines.NotificationListener;
+import org.kie.internal.task.api.UserInfo;
+import org.kie.internal.task.api.model.NotificationEvent;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CountDownLatchNotificationListener implements NotificationListener {
+
+    private CountDownLatch latch;
+    private static List<NotificationEvent> eventsReceived = new ArrayList<NotificationEvent>();
```

Comment:
This shouldn't be static, and it's initialized at constructor (I think that's the reason of test failure at CI);
```suggestion
    private List<NotificationEvent> eventsReceived;
```

Reply:
Done, waiting for Jenkins to rebuild